mlir/lib/Dialect/Rock/IR/RockDialect.cpp

```diff
@@ -3113,21 +3124,39 @@ AttnPerfConfigAttr AttnPerfConfigAttr::get(StringAttr perfConfigStrAttr) {
     llvm::to_integer(s, param);
     return param;
   });
+  bool isV3 = version == 3;
```

Comment:
nit: for better readability: 
```suggestion
  bool isV3 = (version == 3);
```